docs/docs/jsonnet/native-functions.md

```diff
@@ -6,17 +6,51 @@ menu: "Writing Jsonnet"
 
 # Native Functions
 
-Tanka extends the Jsonnet standard library with some common helper utilities.
+Tanka extends Jsonnet using _native functions_, offering additional functionality not yet available in the standard library.
 
-## parseJSON
+To use them in your code, you need to access them using `std.native` from the standard library:
+
+```jsonnet
+{
+  deployment:  std.native('<name>')('<arguments>'),
+}
+```
+
+`std.native` takes the native function's name as a `string` argument and returns a `function`, which is called using the second set of parentheses.
+
+## parseJson
+
+### Signature
 
 ```ts
 parseJSON(string json) Object
 ```
 
-`parseJSON` wraps `json.Unmarshal` to convert a json string into a dict
+`parseJSON` wraps `json.Unmarshal` to convert a json string into a dict.
```

Comment:
```suggestion
`parseJSON` parses a json string and returns the respective Jsonnet type (`Object`, `Array`,  etc).
```

Thinking about this, nobody cares which underlying Go function  is used, so let's describe the behaviour instead

Reply:
Agreed